cli/core.go

```diff
@@ -67,6 +67,7 @@ var commands = []Command{
 	{"failover", nil, failover},
 	{"override", &commandOverride, nil},
 	{"show", &commandShow, nil},
+	{"help", nil, help},
```

Comment:
I'd move this up to keep sorted (although "quit" is also in the incorrect place).